src/phone-number-input/phone-number-input.tsx

```diff
@@ -25,26 +23,29 @@ export const PhoneNumberInput = ({
     // =============================================================================
     // CONST, STATE, REF
     // =============================================================================
-    const [inputNumber, setInputNumber] = useState(value?.number || "");
-    const [selectedCountryCode, setSelectedCountryCode] = useState(
-        value?.countryCode || ""
-    );
+    const options = useMemo(() => PhoneNumberInputHelper.getCountries, []);
```

Comment:
Well it is actually a function since it does a map. I can convert it to be a function to be more aligned and skip the memoisation. But I think maybe adding to the state might be better else the function will keep running upon re-render